starter-code/css/main.css

```diff
@@ -7,5 +7,57 @@ light blue: #1c5380
 body {
```

Comment:
Try something like this using the bgtile.png:

body {
  background: #fff url('../images/bgtile.png') repeat-x;
  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
  margin: 0;
}